docs/logic-apps/control-single-logicapp.md

```diff
@@ -46,9 +46,11 @@ using (var logicApp = ...)
 }
 ```
 
-### Temporary enable Logic App
+### Temporary enable/disable Logic App
```

Comment:
Might be good to include a small list of features on https://github.com/invictus-integration/testing-framework/pull/67/files#diff-5abb4c01eb88a417659d0f7429e0a709R17 maybe? or we change all ### to ## so that they appear in the subnavigation? (but feel free to create dedicated issue/PR for this)

Reply:
Ah, I see, like we did on other Arcus doc pages.